src/plugins/bibliography/bibliography-utils.ts

```diff
@@ -170,12 +170,25 @@ const getLatest = (a: TrackedAttrs, b: TrackedAttrs) =>
  * Map PM node(bibliography, citation) to Model and it could be map by dataTracked if it's exist
  * as it's easier to deal with the manuscript Models for both references list & citation popup view
  */
-const getEffectiveAttrs = (node: ManuscriptNode): Attrs | undefined => {
+const getEffectiveAttrs = (
+  node: ManuscriptNode,
+  excludeDeletedNode?: boolean
+): Attrs | undefined => {
   const { dataTracked, ...attrs } = node.attrs
   const nodeChange = (dataTracked as TrackedAttrs[] | undefined)?.reduce(
     getLatest
   )
 
+  const isDeleted =
+    nodeChange &&
+    nodeChange.operation === CHANGE_OPERATION.delete &&
+    nodeChange.status !== CHANGE_STATUS.rejected
```

Comment:
Can we use utils from `track-changes-utils.ts` here?

Reply:
We maybe could. The only reason I hesitate is due to the use of `.some()` in that function, which is a bit confusing to me. In that function, it searches for any change that matches the condition, rather than specifically targeting the last change.

Anyways, I can't think of a scenario where it could break if we use `isDeleted` from `utils`. Do you have anything in mind?